safe_autonomy_sims/gym/inspection/reward.py

```diff
@@ -219,13 +220,18 @@ def facing_chief_reward(chief: sim.Target, deputy: sim.Inspector, epsilon: float
     float
         reward value
     """
+    reward = 0.0
     rel_pos = chief.position - deputy.position
     rel_pos = rel_pos / np.linalg.norm(rel_pos)
-    gaussian_decay = np.exp(-np.abs(((np.dot(
+
+    diff = (np.dot(
         Rotation.from_quat(deputy.camera.orientation).as_euler("XYZ"),
         rel_pos,
-    ) - 1)**2) / epsilon))
-    reward = 0.0005 * gaussian_decay
+    ))**2
+    abs_diff = abs(diff)
+    if not abs_diff > max_diff:
```

Comment:
Why is this condition checked?

Reply:
That is an excellent question. This facing chief reward is the final thorn in my side regarding aligning the 6dof environments. Everything else in the validation test passes, except for this reward. CoRL's facing chief reward seems to use this GaussianDecayFromTargetValue reward to wrap the facing chief dot product. You can see the check [here](https://github.com/act3-ace/safe-autonomy-sims/blob/main/safe_autonomy_sims/rewards/cwh/gaussian_decay_from_target_value.py#L98) and the configuration of it [here](https://github.com/act3-ace/safe-autonomy-sims/blob/main/configs/weighted-six-dof-inspection/agent.yml#L254). Clearly, since the rewards are still not align, I'm doing something wrong here. But it's proven difficult to find exactly what.